### PR TITLE
Update README.md inline with other Azure actions

### PR DESCRIPTION
**Pull request by pulkitaggarwl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,35 @@
 ![Integration Test](https://github.com/Azure/aml-registermodel/workflows/Integration%20Test/badge.svg?branch=master&event=push)
 ![Lint and Test](https://github.com/Azure/aml-registermodel/workflows/Lint%20and%20Test/badge.svg?branch=master&event=push)
 
-# Azure Machine Learning Register Model Action
+# GitHub Action for Registering a Machine Learning Models with Azure
```
Comment (marvinbuss) [inline README.md]: Typo

**Review by marvinbuss — COMMENTED** [review]
Please fix the typo

**Review by awmatheson — APPROVED** [review]
:+1:

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**